app/_src/gateway/production/networking/configure-tls.md

```diff
@@ -0,0 +1,252 @@
+  * Encryption: The data that is transmitted between {{site.base_gateway}} and the PostgreSQL server is encrypted using symmetric key encryption, this means the same key used to encrypt the data is used to decrypt the data. This method protects your data from unauthorized access. 
+  * Data integrity: With TLS your data is protected during transport.
+* Disadvantages: 
+  * Complexity: Configuring and maintaining secure TLS connections require the management of certificates and may add additional complexity to your {{site.base_gateway}} instance.
+  * Performance: Encrypting and decrypting data consumes additional computing resources which can impact the performance of {{site.base_gateway}}. 
+
+This guide will show you how to configure TLS on PostgreSQL and {{site.base_gateway}} to secure your data in transport. 
+
+
+## Prerequisites <!-- Optional -->
+
+* [OpenSSL](https://www.openssl.org/)
+* TLS support enabled in PostgreSQL
+  * If you are installing a pre-packaged distribution, or using the official docker image, TLS support is already compiled in.
+  * If you are compiling PostgreSQl from source, TLS support must be enabled at build time using the `--with-ssl=openssl ` flag. In this case OpenSSL will need to be installed as well. 
+
+## PostgreSQL setup
+
+To setup Postgres server with TLS enabled you have to configure the following parameters in the `postgresql.conf` file:
+
+
+```bash
+ssl = on #This parameter tells PostgreSQL to start with `ssl`. 
+ssl_cert_file = '/path/to/server.crt' # This is the default directory, other locations and parameters can be specified. 
+ssl_key_file = '/path/to/server.key' # This is the default directory, other locations and parameters can be specified. 
+```
+
+With these settings the server will establish a secure communication channel with a client using TLS, and if the client supports TLS proceed with the TLS handshake and establish a secure connection. In the case where a client does not support SSL or TLS the system will fall back on an unsecured connection. 
+
+The key files that are referenced in the PostgreSQL configuration must have the right permissions. If the file is owned by the database user, it must be set to `u=rw (0600)`, you can set this using the following command: 
+
+`chmod 0600 /path/to/server.crt`
+
+If the file is owned by the root user, the permissions for the file must be `u=rw,g=r (0640)`, you can set this using the following command: 
+
+`chmod 0640 /path/to/server.crt`. 
+
+Certificates issued by intermediate certificate authorities can also be used, but the first certificate in `server.crt` must be the server's certificate, and it must match the server's private key. This is because the certificate must match the private key of the server, and the intermediate certificate authority to establish a chain of trust. 
```

Comment:
```suggestion
Certificates issued by intermediate certificate authorities can also be used, but the first certificate in `server.crt` must be the server's certificate, and it must match the server's private key. This is because the certificate must match the server's private key and the intermediate certificate authority, to establish a chain of trust. 
```